src/Kathara/cli/ui/setting/KubernetesOptionsHandler.py

```diff
@@ -153,7 +153,44 @@ def add_items(self, current_menu: ConsoleMenu, menu_formatter: MenuFormatBuilder
 
         image_pull_policy_item = SubmenuItem(image_pull_policy_string, image_pull_policy_menu, current_menu)
 
+        # Private Registry dockerconfigjson Option
+        pr_dcj_string = "Private Registry Docker Configuration"

```

Comment:
The name of the variable `pr_dcj_*` is not understandable. 
Please use a proper readable name, for example `docker_private_registry_*`

Reply:
I would choose the name `docker_config_json_*` since the string would be a base64 encoding of the content of `~/.docker/config.json`. This way the naming stays consistent with the one used in the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials) with just some `_` added for better readability.